### PR TITLE
Removed UI-focused user docs from AWX.

### PR DESCRIPTION
**Pull request by tvo318**
##### SUMMARY
Removed user docs with UI-focus from AWX as part of the AWX modernization efforts. 

##### ISSUE TYPE
 - Bug, Docs Fix or other nominal change

##### COMPONENT NAME
 - Docs

##### AWX VERSION
latest

### CODE REVIEW

**Comment by oraNod** [comment]
Hey @tvo318 Looks like there is some funky indentation in the left nav:

image

**Comment by oraNod** [comment]
I think the left nav is off because "Release Notes" is an external link instead of a local RST file. could you try creating a file called something like `release_notes.rst` that has this in it?

```
Release Notes
============
.. raw:: html
   <meta http-equiv="refresh" content="0; url=https://github.com/ansible/awx/releases">
```

then put that in the toctree before `known_issues` like this:

```
.. toctree::
   :maxdepth: 2
   :caption: Welcome

   release_notes
   known_issues
```

**Review by oraNod — APPROVED** [review]
LGTM. Thank you @tvo318

**Review by AlanCoding — APPROVED** [review]

**Review by gundalow — COMMENTED** [review]
Have you checked the build log for errors?
I was expecting there to be more `:ref:` updates to remove links to pages that no longer exist

**Comment by tvo318** [comment]
@gundalow - I removed all xrefs that would produce broken links and removed/modified them accordingly.

File: docs/docsite/conf.py [hunk 1]
```diff
@@ -8,10 +8,10 @@
 sys.path.insert(0, os.path.abspath('./rst/rest_api/_swagger'))
 
 project = u'Ansible AWX'
-copyright = u'2023, Red Hat'
+copyright = u'2024, Red Hat'
```
Comment (samccann) [inline docs/docsite/conf.py:11]: RH legal said we don't need to have a date in the copyright statement.  So you can pull 2024 out of this and never have to edit it again! Also if this is opensource, we may want a more generic owner. like copyright = "AWX project contributors"

File: docs/docsite/conf.py [hunk 2]
```diff
@@ -8,10 +8,10 @@
...
 author = u'Red Hat'
 
-pubdateshort = '2023-08-04'
+pubdateshort = '2024-11-22'
 pubdate = datetime.strptime(pubdateshort, '%Y-%m-%d').strftime('%B %d, %Y')
```
Comment (samccann) [inline docs/docsite/conf.py:15]: you could experiment with removing this pubdate info and pubdateshort with something like [this](https://github.com/ansible/ansible-documentation/blob/devel/docs/docsite/rst/conf.py#L288) which we use on the core/package docs... and ... never update it again :-)

**Review by samccann — APPROVED** [review]
LGTM. A couple of conf.py suggestions you could consider.

**Review by blkingrh — APPROVED** [review]

**Review by jillr — APPROVED** [review]

**Review by gundalow — APPROVED** [review]

File: docs/docsite/conf.py (hunk 1 above)
Comment (tvo318) [reply docs/docsite/conf.py:11]: Let's do this is in a subsequent PR.

File: docs/docsite/conf.py (hunk 2 above)
Comment (tvo318) [reply docs/docsite/conf.py:15]: As agreed, let's do this is in a subsequent PR.

**Comment by FaxMac** [comment]
Why was the Quickstart and Administration sections deleted?

**Comment by oraNod** [comment]
> Why was the Quickstart and Administration sections deleted?

Hi @FaxMac those sections were removed from the docs to reflect changes in the codebase that have been happening as part of the ongoing effort to move AWX towards a more modern services-oriented architecture. You can find all the details about that effort in the forum. Please check out the posts with the `awx-modernization` tag here: https://forum.ansible.com/tag/awx-modernization

The docs for the last AWX release, version 24.6.1, are still available and include the Quickstart and Administration sections. You can find those here: https://ansible.readthedocs.io/projects/awx/en/24.6.1/

**Pull request merged**